test/demo/activation-keys.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  tasks:
+  - name: "Create katello client activation key"
+    katello_activation_key:
+      username: "admin"
```

Comment:
I don't want to overly complicate this at first. It's definitely something we could easily add in the future, but these values are really just example values.